src/views/profile.hbs

```diff
@@ -0,0 +1,106 @@
+<div id="modal-details" class="modal">
+  <div class="modal-background">
+    {{> profile_myDetails }}
+  </div>
+</div>
+
+
+
+<nav id="navigation">
+  <a href="/allusers" class="" id="">ALL USERS</a>
+  <a href="" class="" id="">MY PROFILE</a>
+</nav>
+
+<header>
```

Comment:
Perhaps consider splitting the profile view components (`header` and `article`) into a separate `.hbs` file.